providers/standard/src/airflow/providers/standard/operators/hitl.py

```diff
@@ -84,6 +84,9 @@ def __init__(
         self.multiple = multiple
 
         self.params: ParamsDict = params if isinstance(params, ParamsDict) else ParamsDict(params or {})
+        if hasattr(ParamsDict, "filter_params_by_source"):
+            # Params that exist only in Dag level does not make sense to appear in HITLOperator
+            self.params = ParamsDict.filter_params_by_source(self.params, source="task")
```

Comment:
Should we add debug log here so when there's no filter_params_by_source, users are told about it:

```python
else:
    log.debug("ParamsDict.filter_params_by_source not available; HITLOperator will show all params.")
```